packages/client/hooks/useSpotlightSimulatedDrag.tsx

```diff
@@ -54,9 +54,6 @@ const useSpotlightSimulatedDrag = (
       const meetingProxy = store.get(meetingId)
       meetingProxy?.setValue(null, 'spotlightGroup')
       meetingProxy?.setValue(null, 'spotlightReflectionId')
-      const reflection = store.get(reflectionId)
```

Comment:
Looks like this is redundant code now. Removing this ensures that the source reflection returns to the kanban immediately once the Spotlight modal closes